lib/index.js

```diff
@@ -1,13 +1,25 @@
-module.exports = {
+import fs from 'fs'
+
+const pkg = JSON.parse(fs.readFileSync(new URL('../package.json', import.meta.url), 'utf8'))
```

Comment:
Can't we simply import it here?


```suggestion
import pkg from '../package.json'
```

Reply:
This was taken from eslint plugin creation guide and it worked. Will rather leave it like this